File: src/desktop/Models/Project.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace Transleet.Desktop.Models;

public class Project
{
    public Guid Id { get; set; }
    public string? DisplayName { get; set; }
    public string? Name { get; set; }
    public string? Avatar { get; set; }
    public string? Description { get; set; }
    public DateTimeOffset CreatedAt { get; set; }
    public DateTimeOffset UpdatedAt { get; set; }
    public ICollection<Term>? Terms { get; set; }
    public ICollection<Component>? Components { get; set; }
    public short? Status { get; set; }
    public short? AccessLevel { get; set; }
    public bool? Hide { get; set; }
    public Organization? Organization { get; set; }


}

File: src/svelte/src/lib/api/models/Project.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { Component } from './Component';
import type { Organization } from './Organization';
import type { Term } from './Term';

export type Project = {
    id?: string;
    displayName?: string | null;
    name?: string | null;
    avatar?: string | null;
    description?: string | null;
    createdAt?: string;
    updatedAt?: string;
    terms?: Array<Term> | null;
    components?: Array<Component> | null;
    status?: number | null;
    accessLevel?: number | null;
    hide?: boolean | null;
    organization?: Organization;
};